content/posts/2022/retention-rules.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Automatically delete workspace files with custom file retention rules"
+date: 2022-12-07T10:30:00+01:00
```

Comment:
Since the deployment will be at 08:00 CET, please put the blog post hour to 05:00.

Also, WRT slug, perhaps we can go for `/2022/workspace-file-retention-rules` to make sure to that retention rules are related to workspace files.